Bottom: b9fa66f6d46bc9048c98647639cf98e66c834deb
Top:    2e8d4441f6d3087b6a0041935ae5aff2f704aebc
Author: Jon Smirl <jonsmirl@gmail.com>
Date:   2012-04-12 22:36:27 -0400

Refresh of dma

---

diff --git a/arch/arm/boot/dts/ea3131.dts b/arch/arm/boot/dts/ea3131.dts
index 3320ef3..0911d76 100644
--- a/arch/arm/boot/dts/ea3131.dts
+++ b/arch/arm/boot/dts/ea3131.dts
@@ -35,8 +35,8 @@
 	spi@15002000 {
 		gpios = <&gpio_spi 4 0 
 			 &gpio_ebi_i2stx_0 3 0>; 
-		s25sl032p@0 {
-			compatible = "code,s25sl032p";
+		s25sl032a@0 {
+			compatible = "code,s25sl032a";
 			reg = <0>;
 			spi-max-frequency = <1000000>;
 		};
diff --git a/drivers/spi/spi-lpc313x.c b/drivers/spi/spi-lpc313x.c
index baeca08..5a982c7 100644
--- a/drivers/spi/spi-lpc313x.c
+++ b/drivers/spi/spi-lpc313x.c
@@ -11,8 +11,8 @@
  */
 
 #define DEBUG
-#define jds_printk printk
-//#define jds_printk(format, arg...) ({if (0) printk(format, ##arg);})
+//#define jds_printk printk
+#define jds_printk(format, arg...) ({if (0) printk(format, ##arg);})
 
 #include <linux/io.h>
 #include <linux/clk.h>
@@ -544,17 +544,8 @@ static int lpc31xx_spi_transfer(struct spi_device *spi, struct spi_message *msg)
  */
 static void lpc31xx_spi_cleanup(struct spi_device *spi)
 {
-	//struct lpc31xx_spi_chip *chip;
 	jds_printk("JDS - lpc31xx_spi_cleanup\n");
-#if 0
-	chip = spi_get_ctldata(spi);
-	if (chip) {
-		if (chip->ops && chip->ops->cleanup)
-			chip->ops->cleanup(spi);
-		spi_set_ctldata(spi, NULL);
-		kfree(chip);
-	}
-#endif
+	spi_set_ctldata(spi, NULL);
 }
 
 /**
